common/utils/src/main/resources/error/error-classes.json

```
@@ -1,4 +1,32 @@
 {
+  "AGGREGATE_FILTER_EXPRESSION_ERROR" : {
+    "message" : [
+      "FILTER expression in aggregate function has errors:"
+    ],
+    "subClass" : {
+      "CONTAINS_AGGREGATE" : {
+        "message" : [
+          "FILTER expression contains an aggregation."
```

Let's tell to users what we expected and what we found:
```suggestion
          "Expected an expression without any aggregation but found <aggExpr>."
```

---

And if you can, point out the aggregate expression. Just take `AggregateExpression` and pass it to `aggregateInAggregateFilterError()` in:
```scala
            case Some(filter) if filter.exists(_.isInstanceOf[AggregateExpression]) =>
              throw QueryCompilationErrors.aggregateInAggregateFilterError()
```

---

Done as suggested.